### PR TITLE
Add recreate strategy to postgres

### PR DESCRIPTION
**Pull request by danielpassos**
See: [AEROGEAR-8778](https://issues.jboss.org/browse/AEROGEAR-8778)

### CODE REVIEW

**Comment by danielpassos** [comment]
@wei-lee, @darahayes Can you double check it solve the problem before I send for the others APBs? cc/ @grdryn

**Comment by darahayes** [comment]
Checking now 👀

File: roles/provision-unifiedpush-apb/tasks/provision-database.yml [hunk 1]
```diff
@@ -42,6 +42,10 @@
     name: '{{ postgres_deploymentconfig_name }}'
     namespace: '{{ namespace }}'
     replicas: 1
+    spec_strategy_type: 'Recreate'
+    spec_strategy_active_deadline_seconds: 21600
+    spec_strategy_recreate_params:
+      timeoutSeconds: 600
```
Comment (wei-lee) [inline roles/provision-unifiedpush-apb/tasks/provision-database.yml]: can we move these values to be parameters so that it can be overridden

**Review by wei-lee — APPROVED** [review]
small suggestion on use parameters for some of the values, otherwise it has been verified on a cluster.

**Comment by darahayes** [comment]
I've just verified that it works. Once we address the feedback, I think this is good to merge!

**Comment by danielpassos** [comment]
@wei-lee, @darahayes Done!

**Review by wei-lee — APPROVED** [review]

**Review by psturc — APPROVED** [review]

**Pull request merged**